src/ui/public/vis/editors/default/default.js

```diff
@@ -2,11 +2,12 @@ import './sidebar';
 import './vis_options';
 import $ from 'jquery';
 
-
 import _ from 'lodash';
 import angular from 'angular';
 import defaultEditorTemplate from './default.html';
 
+import { VisEditorTypesRegistryProvider } from 'ui/registry/vis_editor_types';
+
```

Comment:
this might be problematic .... 
right now someone might import the default editor (or any of the other things registered) into his code, which will register a duplicate with registryProvider ? (not sure does registryProvider allow duplicates ? what will happen in that case?) should either check for this scenario, or not export anything (so importing of this module is not possible, it should only be accessed thru registry